AirCasting/Settings/SettingsView.swift

```diff
@@ -10,23 +10,91 @@ import Foundation
 
 struct SettingsView: View {
     let logoutController: LogoutController
-
+    @State private var isToggle : Bool = false
```

Comment:
I know it's nit-picky, but the usual styling (in the app too afaict) is to put a colon after the name without any spaces:

```Swift
@State private var isToggle: Bool = false
```